docs/transpile/custom-backend.ipynb

```diff
@@ -39,7 +39,11 @@
     "\n",
     "The `BackendV2` object is an abstract class used for all backend objects created by a provider (either within `qiskit.providers` or another library such as [`qiskit_ibm_runtime.IBMBackend`](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.IBMBackend)).  As mentioned above, these objects contain several attributes, including a [`Target`](/api/qiskit/qiskit.transpiler.Target). The `Target` contains information that specifies the backend's attributes - such as the [`Coupling Map`](/api/qiskit/qiskit.transpiler.CouplingMap), list of [`Instructions`](/api/qiskit/qiskit.circuit.Instruction), and others - to the transpiler.  In addition to the `Target`, one can also define pulse-level details such as the [`DriveChannel`](/api/qiskit/qiskit.pulse.channels.DriveChannel) or [`ControlChannel`](/api/qiskit/qiskit.pulse.channels.ControlChannel).\n",
     "\n",
-    "The following example demonstrates this customization by creating a simulated multi-chip backend, where each chip possesses a heavy-hex connectivity.  The example specifies the backend's two-qubit gate set to be [`CZGates`](../api/qiskit/qiskit.circuit.library.CZGate) within each chip and [`CXGates`](../api/qiskit/qiskit.circuit.library.ECRGate) between chips.  First, create your own `BackendV2` and customize its `Target` with single and two-qubit gates according to the previously described constraints."
+    "The following example demonstrates this customization by creating a simulated multi-chip backend, where each chip possesses a heavy-hex connectivity.  The example specifies the backend's two-qubit gate set to be [`CZGates`](../api/qiskit/qiskit.circuit.library.CZGate) within each chip and [`CXGates`](../api/qiskit/qiskit.circuit.library.ECRGate) between chips.  First, create your own `BackendV2` and customize its `Target` with single and two-qubit gates according to the previously described constraints.\n",
+    "\n",
+    "<Admonition type=\"tip\" title=\"`graphviz`\">\n",
+    "Install the `graphviz` library before trying the following code. It is required to plot a coupling map.\n",
```

Comment:
```suggestion
    "Plotting a coupling map requires the `graphviz` library to be installed.\n",
```

Reply:
Ha, I was trying to avoid the gerund but had come up with the same sentence :D I'll go with your suggestion since it was my first thought too